examples/src/Viewer.tsx

```diff
@@ -702,6 +702,16 @@ class Viewer extends React.Component<InputParams, ViewerState> {
         this.setState({ isRecording: isRecording });
     };
 
+    // Remove the Viewer's onRecordedMovie prop assigment to disable feature
+    private isRecordingEnabledInViewer = () => {
+        const viewerElement = this.viewerRef.current;
+        if (viewerElement) {
+            const viewerProps = viewerElement.props;
+            return !!viewerProps.onRecordedMovie;
```

Comment:
again, I know this is the test app, but it's to demo the feature. If I was installing this app, I wouldn't test my own prop to see if I was going to enable a feature. I would just have a boolean button that toggled  a feature. So here, to demo this I would make a "disable/enable recording" button that changes the app state. and when that changes I would change the `onReordedMovie` prop to null and wouldn't render the RecordMovieCoponent` at all, instead of adding a prop. that way it's mimicking the way we would use the viewer

Reply:
Ok, that sounds good